torch/_inductor/codegen/cpp_gemm_template.py

```diff
@@ -121,9 +119,15 @@
                         {{ micro_gemm.codegen_call(kernel, tile_X, tile_W, acc, accum=True)|indent(24, false) }}
                     }
                 }
-                {%- set tile_Y = kernel.slice_nd(Y_2d, [("m_start", "m_end"), ("n_start", "n_start + N0")]) %}
+                {%- if N == PADDED_N %}
+                    {%- set tile_Y = kernel.slice_nd(Y_2d, [("m_start", "m_end"), ("n_start", "n_start + N0")]) %}
+                    {%- set tile_acc = acc %}
```

Comment:
Feel like this line can also be `{%- set tile_acc = kernel.slice_nd(acc, [(), ("0", "n_end - n_start")]) %}`? For no_paddeding case, will it be same as `acc`?

Reply:
ditto